docs/Quantiles/QuantilesOverview.md

```diff
@@ -79,7 +79,8 @@ These sketches have many parallel methods. Please refer to the sketch API docume
 		* Inclusive, a common definition in some of the theoretical literature.
 	* Accuracy %: a function of *K* and independent of rank. 
 	* Near optimal accuracy per sketch size compared to other constant accuracy quantiles sketches. 
-	* Java: Dedicated *float* and *double* implementations.
+	* Java: Dedicated *float*, *double*, *long*, and generic *item* implementations.
```

Comment:
i generally favor `double` rather than *double* for types. Backticks rather than asterisks. That's what, say, javadoc typically uses for types, too.